code/game/verbs/who.dm

```diff
@@ -68,9 +68,6 @@
 						entry += " - <font color='black'><b>DEAD</b></font>"
 				else
 					entry += " - <font color='black'><b>DEAD</b></font>"
-		if(is_special_character(C.mob))
-			entry += " - <b><font color='red'>Antagonist</font></b>"
-	entry += " (<A HREF='?_src_=holder;adminmoreinfo=\ref[C.mob]'>?</A>)"
```

Comment:
why did you remove the ? href too, and btw i don't think this really needs to be removed either